zebra-chain/src/parameters/network/subsidy.rs

```diff
@@ -379,6 +386,15 @@ pub trait ParameterSubsidy {
     ///
     /// [7.10]: <https://zips.z.cash/protocol/protocol.pdf#fundingstreams>
     fn height_for_first_halving(&self) -> Height;
+
+    /// Returns the halving interval after Blossom
+    fn post_blossom_halving_interval(&self) -> HeightDiff;
+
+    /// Returns the halving interval before Blossom
+    fn pre_blossom_halving_interval(&self) -> HeightDiff;
+
+    /// Returns the address change interval for funding streams
```

Comment:
The [reference to the spec here is also wrong/outdated](https://github.com/ZcashFoundation/zebra/blob/regtest-halving-interval/zebra-chain/src/parameters/network/subsidy.rs#L317-L321) if we could update it to refer to 7.10 instead of 7.10.1 there too.

```suggestion
    /// Returns the address change interval for funding streams
    /// as described in [protocol specification §7.10][7.10].
    ///
    /// > FSRecipientChangeInterval := PostBlossomHalvingInterval / 48
    ///
    /// [7.10]: https://zips.z.cash/protocol/protocol.pdf#zip214fundingstreams
```